### PR TITLE
Fix encoding issues in Spanish legend texts

### PR DESCRIPTION
**Pull request by bryant1410**

### CODE REVIEW

File: core/src/main/resources/jenkins/model/Jenkins/legend_es.properties [hunk 1]
```diff
@@ -20,17 +20,17 @@
...
-red_anime=La �ltima ejecuci�n fall�. Se ha lanzado una nueva ejecuci�n.
+red=La \u00FAltima ejecuci\u00F3n fall\u00F3.
+red_anime=La \u00FAltima ejecuci\u00F3n fall\u00F3. Se ha lanzado una nueva ejecuci\u00F3n.
 
-health-81plus=El estado de salud del proyecto supera el 80%. Puedes situar el rat�n sobre el icono para ver una explicaci�n detallada.
-health-00to20=El estado de salud del proyecto es inferior al 20%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
-health-41to60=El estado de salud del proyecto est\u00E1 entre el 40% y el 60%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
-health-21to40=El estado de salud del proyecto est\u00E1 entre el 20% y el 40%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
-health-61to80=El estado de salud del proyecto est\u00E1 entre el 80% y 80%. Puedes situar el rat\u00F3n sobre el icono para ver una explicaci\u00F3n detallada.
+health-81plus=El estado de salud del proyecto supera el 80%. Puedes situar el ratón sobre el \u00EDcono para ver una explicaci\u00F3n detallada.
```
Comment (amuniz) [inline core/src/main/resources/jenkins/model/Jenkins/legend_es.properties]: _icono_ is not accented.

Comment (bryant1410) [inline core/src/main/resources/jenkins/model/Jenkins/legend_es.properties]: Ok, [I see that some people say it without accent](https://es.wikipedia.org/wiki/Icono). Will change

Comment (bryant1410) [inline core/src/main/resources/jenkins/model/Jenkins/legend_es.properties]: Seems to be people from Spain according to this: http://www.alegsa.com.ar/Diccionario/C/4074.php

Comment (amuniz) [inline core/src/main/resources/jenkins/model/Jenkins/legend_es.properties]: Oh, you are right http://dle.rae.es/?id=KsRzX3u `ícono` is accepted too (only in America). I've never seen it accented BTW (even in LATAM writings).

**Comment by oleg-nenashev** [comment]
@amuniz Are you fine with it?

**Comment by daniel-beck** [comment]
CC @recena @carlossg @andresrc

**Comment by daniel-beck** [comment]
@bryant1410 What exactly is this fixing? All the chars are in Latin-1, so fine for properties files…

**Comment by bryant1410** [comment]
@daniel-beck well, it all started when I found an error in the Spanish translations (#2535), and then realized that some chars weren't able to be displayed by vim and saw that others are encoded with their Unicode number. So I changed all to be the same.

**Comment by daniel-beck** [comment]
This change is not actually necessary as explained above, and actually makes reading the file in GitHub/editors not automatically resolving Unicode escapes more difficult, so -0.5 from me.

Any other opinions?

**Comment by recena** [comment]
I agree with @bryant1410, if the file contains Unicode escapes, we should use it in the file.

At any way, please, file a JIRA issue.

**Comment by bryant1410** [comment]
I mean, I prefer to just use the chars in utf-8, but all the same way.

You mean me with the JIRA thing?

**Comment by recena** [comment]
@bryant1410 My general recommendation is to provide changes always associated to a JIRA issue.

**Comment by bryant1410** [comment]
@recena but should I fill the JIRA issue? Or you guys?

**Comment by oleg-nenashev** [comment]
If it is not a subject for backporting, JIRA issue is not required IMHO. And we do not backport translations unless there is something horrible there

**Comment by oleg-nenashev** [comment]
@recena Are you fine with going forward without an issue? Not a subject for backporting

**Comment by oleg-nenashev** [comment]
@recena ping

**Comment by oleg-nenashev** [comment]
Merging since there is no response from @recena and since it's not a candidate for backporting

**Comment by oleg-nenashev** [comment]
Well, noticed -0.5 from @daniel-beck . No merge right now, needs Daniel's feedback

**Comment by oleg-nenashev** [comment]
@daniel-beck WDYT? Just to close/merge this PR in whatever way

**Comment by daniel-beck** [comment]
AFAICT there are no positive reviews. As I consider this change to be unnecessary, closing this for now.

**Comment by recena** [comment]
I'm so sorry, I was a bit busy for the last weeks.

**Pull request closed without merging**